docs/clearlinux-installation-guide.md

```diff
@@ -0,0 +1,52 @@
+# Installing Clear Containers 3.0 on Clear Linux
+
+Clear Containers 3.0 is available in Clear Linux since version **17890**.
+
+*Note:
+If you already have a previous Clear Linux version with Clear Containers 2.x installed,
```

Comment:
This sentence starts right after the Note:. Add a blank line after Note: to have the sentence start on its own line.